recipes/OceanColor/meta.yaml

```
@@ -0,0 +1,55 @@
+{% set name = "OceanColor" %}
+{% set version = "0.0.9" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/OceanColor-{{ version }}.tar.gz
+  sha256: 078aa5aec1dc61ce560196795ef236586aa4e1cf289de27cb3a8750849f7038f
+
+build:
+  number: 0
+  noarch: python
+  entry_points:
+    - OceanColor=OceanColor.cli:main
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - pytest-runner
+    - python >=3.6
+  run:
+    - click >=7.1.1 numpy>=1.17 h5netcdf>=0.8 h5py>=2.10 tables>=3.6 netcdf4>=1.5 pandas>=1.0 pyproj>=2.6 requests>=2.23 xarray>=0.16
```

Looks like something left over?
```suggestion
    - click >=7.1.1
```

---

Ooops. Thanks Ryan!